esp-hal/src/lcd_cam/cam.rs

```diff
@@ -125,14 +126,14 @@ pub struct Cam<'d> {
 /// Represents the camera interface with DMA support.
 pub struct Camera<'d> {
     lcd_cam: PeripheralRef<'d, LCD_CAM>,
-    rx_channel: ChannelRx<'d, <LCD_CAM as DmaEligible>::Dma>,
+    rx_channel: ChannelRx<'d, Blocking, <LCD_CAM as DmaEligible>::Dma>,
 }
 
 impl<'d> Camera<'d> {
     /// Creates a new `Camera` instance with DMA support.
     pub fn new<P, CH>(
         cam: Cam<'d>,
-        channel: ChannelRx<'d, CH>,
+        channel: ChannelRx<'d, Blocking, CH>,
```

Comment:
This combined with the I8080 change has an interesting side effect.

One can no longer, with a single DMA channel, create an async `I8080` and a blocking `Camera`.

Reply:
That is true but temporary

Reply:
I noticed that we're not even relying on DMA channel asyncness in I8080, so instead of a later follow-up, I just changed this PR to take a Blocking channel in both places. I believe this is enough to resolve this issue, especially considering upcoming changes.

Reply:
Yes, that's enough to land this.

Long term, the DMA channel might be made async to await the next descriptor (IN_DONE/OUT_DONE) or await an EOF in streaming transfers.